frontend/components/Task.tsx

```diff
@@ -3,24 +3,31 @@ import { useEffect } from "react";
 
 function Task(props: any): JSX.Element {
```

Comment:
I would avoid using the `any` type. Since you're using Typescript, try to be specific as possible about the type. Serves as good documentation as well.

Reply:
➕